src/modules/dashboard/components/Extensions/ExtensionDetails.css

```diff
@@ -0,0 +1,42 @@
+.main {
+  composes: main from "./Extensions.css";
+}
+
+.content {
+  display: grid;
+  grid-template-columns: auto 35%;
+}
+
+.buttonWrapper {
+  margin-bottom: 10px;
+  text-align: right;
+}
+
+.cellLabel {
```

Comment:
Going by Karol's spec the details are all `13px`, 

I've added a new variable for this, `--size-smallish`, but I think you have to rebase first in order to bring in that change